src/Integrations/Integrations/Laravel/LaravelIntegration.php

```diff
@@ -140,6 +141,20 @@ function (SpanData $span, $args) use ($integration) {
             }
         );
 
+        // used by Laravel >= 5.8
+        // More details: https://laravel.com/docs/5.8/upgrade#events
+        \DDTrace\trace_method(
+            'Illuminate\Events\Dispatcher',
+            'dispatch',
```

Comment:
This needs to allow tracing recursion, I think?

Reply:
Added :-)